frontend/public/style/mixin/_break-word.scss

```diff
@@ -1,6 +1,8 @@
-// Generic mixin for break-word. Note if element is a flex child then min-width: 0% is necessary for it to wrap.
+// Generic mixin for break-word.
```

Comment:
Wonder if it's worth duplicating [the documentation you created in for the previous version of this mixin](https://github.com/openshift/origin-web-common/blob/master/src/styles/_mixins.less#L15-L31) so it's more clear what's what?

Reply:
Added documentation.